File: Iniciante/1010/1010.py
codigo, quantidade, valor = input().split()
codigo = int(codigo)
quantidade = int(quantidade)
valor = float(valor)

codigo2, quantidade2, valor2 = input().split()
codigo2 = int(codigo2)
quantidade2 = int(quantidade2)
valor2 = float(valor2)

resultado = (quantidade * valor) + (quantidade2 * valor2)
print(f'VALOR A PAGAR: R$ {resultado:.2f}')

File: Iniciante/1010/1010.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const linhas = input.split('\n');

const peca1 = linhas[0].split(' ');
const peca2 = linhas[1].split(' ');

const qtd1 = parseInt(peca1[1]);
const valor1 = parseFloat(peca1[2]);

const qtd2 = parseInt(peca2[1]);
const valor2 = parseFloat(peca2[2]);

const total = (qtd1 * valor1) + (qtd2 * valor2);

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);